### PR TITLE
[MM-35039] - Send trial ended email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Adds the assets needed to send the cloud trial ended email on the server

#### Ticket Link
[MM-35039](https://mattermost.atlassian.net/browse/MM-35039?atlOrigin=eyJpIjoiNDgyYjg0Nzg5NzAwNGM1NzhiMmU3ODU5YTkzMmE3MWUiLCJwIjoiaiJ9)

#### Related Pull Requests
- Has server changes (https://github.com/mattermost/mattermost-server/pull/17478)

#### Release Note
```release-note
None
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12784,8 +12784,7 @@
           "version": "3.0.0",
           "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
           "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
-          "dev": true,
-          "optional": true
```
Comment (marianunez) [inline package-lock.json:12788]: Could we revert this unintended changes in the `package-lock.json`?

Comment (AGMETEOR) [reply package-lock.json]: I have reverted the change

**Review by marianunez — APPROVED** [review]

**Comment by AGMETEOR** [comment]
Hey @nickmisasi gentle ping

**Review by nickmisasi — APPROVED** [review]

**Pull request merged**